src/einsteinpy/utils/kerrnewman_utils.py

```diff
@@ -561,3 +563,95 @@ def kerrnewman_time_velocity(pos_vec, vel_vec, mass, a, Q):
     D = (B ** 2) - (4 * A * C)
     vt = (B + np.sqrt(D)) / (2 * A)
     return vt * u.one
+
+
+def event_horizon(
+    M,
+    a,
+    Q,
+    theta=np.pi / 2,
+    coord="BL",
+    c=constant.c.value,
+    G=constant.G.value,
+    Cc=constant.coulombs_const.value,
+):
+    """
+    Calculate the radius of event horizon of Kerr-Newman black hole
+
+    Parameters
+    ----------
+    M : float
+        Mass of massive body
+    a : float
+        Black hole spin factor
+    Q : float
+        Charge on the black hole
+    theta : float
+        Angle from z-axis in Boyer-Lindquist coordinates in radians. Mandatory for coord=='Spherical'. Defaults to pi/2.
+    coord : str
+        Output coordinate system. 'BL' for Boyer-Lindquist & 'Spherical' for spherical. Defaults to 'BL'.
+
```

Comment:
Kindly add the docstrings for `c`,`G` and `Cc`. Please do the same for the other function and in `kerr_utils.py`

Reply:
Okay.